service/src/main/java/com/epam/esm/service/constants/ErrorCodes.java

```diff
@@ -7,6 +7,17 @@
  * @version 1.0
  */
 public class ErrorCodes {
-    public static final int TAG_ERROR_CODE_SUFFIX = 2;
-    public static final int CERTIFICATE_ERROR_CODE_SUFFIX = 1;
+
+    public static final int CERTIFICATE_VALIDATION_EXCEPTION = 42201;
+    public static final int TAG_VALIDATION_EXCEPTION = 42202;
+
```

Comment:
constant order.

More readable if first will be provaded one type of constants, for example for certificat, then all for the tag

Reply:
done